gamebook/scene.py

```diff
@@ -25,9 +28,28 @@ def show_options(self):
         for option in self.options:
             title = option.show_title()
             user_inputs = option.show_user_inputs()
-            nextscene = option.show_next_scene()
+            nextscene = option.show_next_scenes()
             
             choice_list += f"{title} , {user_inputs} , {nextscene}\n"
         
         return choice_list
 
+    def next_scene_draw(self, option_to_evaluate):
```

Comment:
I would maybe change the `option_to_evaluate` to be `next_option_user_input` so it gets the user input directly, no? what do you think?

Reply:
:+1: